frontend/public/components/storage/create-pvc.tsx

```diff
@@ -162,23 +184,29 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
         </label>
         <div className="form-group">
           {this.state.accessModeRadios.map(radio => {
-            const disabled = !!storageClassAccessMode;
-            const checked = !disabled ? radio.value === accessMode : radio.value === storageClassAccessMode;
-            return (
-              <RadioInput
-                {...radio}
-                key={radio.value}
-                onChange={this.handleChange}
-                inline={true}
-                disabled={disabled}
-                checked={checked}
-                aria-describedby="access-mode-help"
-                name="accessMode"
-              />
-            );
+            let radioObj = null;
+            const disabled = storageClassAccessModes.indexOf(radio.value) !== -1 ? false : true;
+
+            storageClassAccessModes.forEach(mode => {
+              const checked = !disabled ? radio.value === accessMode : radio.value === mode;
+              radioObj = (
+                <RadioInput
+                  {...radio}
+                  key={radio.value}
+                  onChange={this.handleChange}
+                  inline={true}
+                  disabled={disabled}
+                  checked={checked}
+                  aria-describedby="access-mode-help"
+                  name="accessMode"
+                />
+              );
+            });
+
+            return radioObj;
           })}
           <p className="help-block" id="access-mode-help">
-            {storageClassAccessMode ? 'Access mode is set by storage class and cannot be changed.' : 'Permissions to the mounted drive.'}
+            Access mode is set by storage class and cannot be changed.
```

Comment:
So it sounds like we are deprecating the access mode that used to be set on the storage class annotation["storage.alpha.openshift.io/access-mode"] since no one used it and it is not documented.  However, storage classes are still not required and therefore we still need more flexibility with this help message since "no storage class" is possible.

Reply:
ack